Tests/scs-compatible-kaas.yaml

```diff
@@ -6,34 +6,30 @@ variables:
 modules:
   - id: cncf-k8s-conformance
     name: CNCF Kubernetes conformance
-    url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0210-v2-k8s-version-policy.md # TODO:!!! this standard mentions cncf but is is the right reference?
+    url: https://github.com/cncf/k8s-conformance/tree/master
     parameters:
-      kubeconfig: relative path to kubeconfig file
+      result_dir_name: 
     run:
       - executable: ./kaas/plugin/run_sonobuoy_executor.py
-        args: -k {subject_root}/{kubeconfig} -r {result_dir_name} -c 'cncf-conformance'
+        args: -k {subject_root}/kubeconfig.yaml -r {subject_root}/{result_dir_name} -c 'cncf-conformance'
```

Comment:
I think `{subject_root}` was meant for input files that are not specific for the run. Output should be generated relative to the current working directory.

Reply:
I see. I would address this by generating directory names similar to the generation of the report output files.

Reply:
I think it would still be cool to have one output directory per subject (just use subdirectories of the temp directory that is being created). But please try to minimize effort here.

Reply:
I agree. I will not address this in this PR and leave the creation of directories to a subsequent issue that does need this handling.